storage/client_test.go

```diff
@@ -498,7 +498,6 @@ func (t *multiTestContextKVTransport) SendNext(done chan<- kv.BatchCall) {
 		done <- kv.BatchCall{Reply: br, Err: nil}
 	}) != nil {
 		done <- kv.BatchCall{Err: roachpb.NewSendError("store is stopped")}
-		t.mtc.expireLeases()
```

Comment:
`multiTestContext` uses a manual clock, so leases will never expire on their own. If a node goes down while holding a lease, we need to expire its leases. This was a crude attempt to do that, by expiring leases whenever the node that DistSender told us to talk to is down. Removing this might make some other tests flaky, but I don't remember which tests. 

In general, tests using multiTestContext haven't been sensitive to spurious lease expirations. Maybe tests where this matters should be using TestCluster instead.